app-backend/tile/src/main/scala/routes/MosaicRoutes.scala

```diff
@@ -21,13 +21,14 @@ import de.heikoseeberger.akkahttpcirce.CirceSupport._
 import cats.data.OptionT
 import cats.implicits._
 
-import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent._
 import scala.util._
 import scala.collection.mutable.ArrayBuffer
 import java.util.UUID
 
-object MosaicRoutes extends LazyLogging with KamonTraceDirectives {
+object MosaicRoutes extends LazyLogging with KamonTrace {
+  val system = AkkaSystem.system
```

Comment:
I removed the implicits with https://github.com/azavea/raster-foundry/pull/2296/commits/4c6567ac2b051bc4b01258b8ed102e1cba348517